src/views/CTLViewsHoC.jsx

```diff
@@ -31,16 +31,20 @@ const CTLViewsHoC = (WrappedView) => {
 		const [paramType, setParamType] = useState(queryParamType.code);
 		const [queryParam, setQueryParam] = useState();
 
```

Comment:
can we change your variable to `idType`?  It shouldn't start with a capital letter.

Also can we make it a const and make the setting of it a return from a function?  

There should also be an addition of a unit test(s) covering the conditions you are checking and the output value given its input.

Reply:
ok will do